### ISSUE TITLE
Existing project with the same name

### ISSUE THREAD

**Issue by hgfischer** [sqle/gitquery#83]
Hello,

How is this project related to https://github.com/cloudson/gitql ?

I guess you did some research before creating this organization and this repository.

**Comment by mcuadros** [comment]
Yep, even we know Cloudson. 
This project is a kind of research for [go-git](https://github.com/src-d/go-git) 

The idea is mainly the same, the big difference is that here we use go-git and the Cloudson's one use git2go.

What is true is that the name is exactly the same, maybe we should rename the project.

**Comment by hgfischer** [comment]
I was working with @cloudson when he created his gitql.

Yes, I think that renaming this project would be fair.

**Comment by smola** [comment]
@hgfischer Thanks for bringing this up.

To clarify further:

> How is this project related to https://github.com/cloudson/gitql ?

The relation is its purpose: SQL + Git, written in Go. There is no relation other than that.

> I guess you did some research before creating this organization and this repository.

Of course. When we started gitql, we wanted to write a SQL layer on top of Git with two main goals:

* Use go-git (pure Go) instead of git2go (libgit2 wrapper) as @mcuadros pointed out.
* Create a standalone SQL engine. Make Git a backend implementation, but make it possible to implement other backends. This engine could be eventually splitted into its own project.

cloudson's gitql did not fit our goals (although it's an awesome project!) and we started from scratch.

We have not given the name too much thought. So we might (or might not) change the name before releasing v1.0.0. We are open to any naming suggestions ;)
